packages/wonder-blocks-dropdown/components/dropdown-launcher.js

```diff
@@ -0,0 +1,244 @@
+// @flow
+
+import * as React from "react";
+import ReactDOM from "react-dom";
+import {StyleSheet} from "aphrodite";
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import Dropdown from "./dropdown.js";
+import type {Item, DropdownItem} from "../util/types.js";
+import ActionItem from "./action-item.js";
+import OptionItem from "./option-item.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /*
+     * The callback function used to open the Dropdown.
```

Comment:
This isn't correct.  It's a render callback which we pass an `openDropdown` function to.  It is responsible for rendering the opener.